### PR TITLE
implemented terms by url

### PR DESCRIPTION
**Pull request by andresgnlez**
- Created `ContentAccordion ` container to allow router management (`push` function).
- When the user clicks on a item, the url will be updated with the new slug entry.
- Back and forward navigation works smoothly.
- Added `slug` property in JSON file: this way we avoid use regexp and other problems generated because inconsistent format. 

The default library didn't include a `componentWillReceiveProps ` function that allows us update the component with new props, that's why we have to create our "own" accordion and overwritting the function to allow us update the component (that allows the accordion being updated if you want to go back to see your las entry).

Probably you will have doubts about this, talk to me if need!

Kudos to Ra for helping me with this!

### CODE REVIEW

**Comment by tiagojsag** [comment]
We need to scroll down automatically to where the open element is

**Comment by andresgnlez** [comment]
Yeah, you are right. Anyway I think we can merge this (if works fine) and develop the scroll apart. This way we ensure having the terms by url feature available anytime.

I'll write the task down on Pivotal.

**Comment by tiagojsag** [comment]
No, implement it in this branch. The current implementation adds nothing to the user. He might see the definition, he might not. The user story is "the user should see the definition", not "the user should see the definition if it's one of the first or his screen is big enough"

**Comment by tiagojsag** [comment]
Seems like the new accordion class is not needed at all - tested like this and it seems to work.
Please add the scroll feature ASAP so we can merge this.

**Comment by andresgnlez** [comment]
With your commit, Accordion won't update if you go back or forward in hsitory navigation. Teh extended function is needed for this functionality.

**Comment by tiagojsag** [comment]
My commit? which commit? I don't see any commit :P
Hadn't tested that scenario. In any case, add the scroll thing please

**Comment by andresgnlez** [comment]
I think the branch's history is quite mess... (it uploaded first commit again..) but I don't see anything strange. Check it out yourself, please.

File: app/src/components/Home/CoverPage.jsx [hunk 1]
```diff
@@ -66,7 +67,7 @@ class CoverPage extends Component {
     const sliderAttributions = [
       '© OCEANA / Juan Cuetos',
       null,
-      '© Steve De Neef ',
+      '© OCEANA / Eduardo Sorensen',
```
Comment (tiagojsag) [inline app/src/components/Home/CoverPage.jsx]: can you change this back,please? the correct value here is the "steve" one

**Comment by tiagojsag** [comment]
once you change the attribution thing, you can insta-merge, no need for another CR

**Pull request merged**